src/AngleSharp.Core.Tests/Html/DOM.cs

```diff
@@ -735,5 +735,15 @@ public void IteratingThroughDomShouldNeverFail_Issue763()
             var html = p.ToHtml();
             Assert.AreEqual(1, img.Count);
         }
+
+        [Test]
+        public void OwnerElement_ShouldNotBeNull()
+        {
+            var document = "<body><div class='one'></div></body>".ToHtmlDocument();
+            var div = document.QuerySelector("div");
+            var attr = div.Attributes["class"];
+
+            Assert.NotNull(attr.OwnerElement);
```

Comment:
Would an additional
```suggestion
            Assert.NotNull(attr.OwnerElement);
            Assert.AreSame(div, attr.OwnerElement);
```
 make sense here?

Reply:
@campersau, done.